src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -19,17 +19,54 @@
  * under the License.
  */
 
+
```

Comment:
not sure why it was previously not following the recommended import order: https://maven.apache.org/developers/conventions/code.html#java-code-convention-import-layouts